integration/basic-fabric.test.ts

```diff
@@ -39,9 +39,9 @@ const execOptions: ExecFileSyncOptionsWithStringEncoding = {
     encoding: "utf-8"
 };
 const versionCombinations: {[version: string]: string[]} = {
-    "2.2.5": ["2.2.5", "1.5.3"],
-    "2.3.3": ["2.3.3", "1.5.3"],
-    "2.4.3": ["2.4.3", "1.5.3"]
+    "2.2": ["2.2.6", "1.5.5"],
```

Comment:
I think the latest version of v2.2 series is 2.2.8 and it should be updated if there is no particular reason.

Reply:
@satota2 Thank you for the review. It is intentional because the fabric-samples repository does not have a tag of v2.2.8, and so the bootstrap script does not work well for v2.2.8.
After it is fixed in the fabric-samples repository, I'd like to post another patch to upgrade the version. Does it make sense?